nixos/modules/tasks/auto-upgrade.nix

```
@@ -80,6 +80,7 @@ in {
           Reboot the system into the new generation instead of a switch
           if the new generation uses a different kernel, kernel modules
           or initrd than the booted system.
+          See `rebootWindow` for configuring the times at which a reboot is allowed.
```

NixOS options currently still need to use DocBook unfortunately :sweat_smile: 

```suggestion
          See <option>rebootWindow</option> for configuring the times at which a reboot is allowed.
```